image.yaml

```diff
@@ -45,6 +45,9 @@ envs:
     - name: "SSO_TRUSTSTORE_SECRET"
       example: "truststore-secret"
       description: "The name of the secret containing the truststore file. Used for volume secretName"
+    - name: "SSO_VAULT_DIR"
+      example: "/etc/sso-vault-secret-volume"
```

Comment:
@martin-kanis Does the `/etc/sso-vault-secret-volume` directory need to exist in the image before / prior the vault feature can be used?

If so, should the `add_vault` routine below, check it exists & throw an error if it doesn't (or forcibly create it, since the particular path was specified via the `SSO_VAULT_DIR` environment variable)

Reply:
No, the vault feature is only turned on and configured if the vault directory exists already

Reply:
Ok, thanks. Requested to add a note to the docs one, so this is clarified there.